helpers/osinfo.go

```diff
@@ -219,3 +220,11 @@ func (btfi *OSInfo) discoverOSDistro() error {
 
 	return nil
 }
+
+func FtraceEnabled() bool {
+	b, err := ioutil.ReadFile("/proc/sys/kernel/ftrace_enabled")
+	if err != nil || len(b) != 1 || b[0] == '0' {
+		return false
+	}
+	return true
+}
```

Comment:
$ echo 1 | sudo tee /proc/sys/kernel/ftrace_enabled
1

$ go run temp.go
false

$ echo 0 | sudo tee /proc/sys/kernel/ftrace_enabled
0

$ go run temp.go
false

Reply:
Something is wrong, no ?

```
[rafaeldtinoco@fujitsu:~]$ cat temp.go
package main

import (
	"fmt"
	"io/ioutil"
)

func FtraceEnabled() bool {
	b, err := ioutil.ReadFile("/proc/sys/kernel/ftrace_enabled")
	c := string(b)
	if c[0] == '0' {
		fmt.Println(false)
	} else if c[0] == '1' {
		fmt.Println(true)
	}

	if err != nil || len(b) != 1 || b[0] == '0' {
		return false
	}
	return true
}

func main() {
	fmt.Println(FtraceEnabled())
}
```
```
[rafaeldtinoco@fujitsu:~]$ go run temp.go
true
false
```

Reply:
Ah woops, there's an EOL appended when reading this way. I'll fix it.

Reply:
Ok it appears to always append the EOL so I fixed it accordingly.